packages/backend-core/src/security/tests/permissions.spec.ts

```diff
@@ -0,0 +1,132 @@
+import { cloneDeep } from "lodash"
+import {
+  doesHaveBasePermission,
+  getBuiltinPermissionByID,
+  isPermissionLevelHigherThanRead,
+  PermissionLevel,
+  PermissionType,
+  levelToNumber,
+  getAllowedLevels,
+  BuiltinPermissionID,
+  getBuiltinPermissions,
+  BUILTIN_PERMISSIONS,
+  Permission,
+} from "../permissions"
```

Comment:
Generally prefer whole module imports  i.e. `* as permissions` so we can just use the single import in all the tests like
```ts
expect(permissions.levelToNumber(...))
```

Reads slightly better in the test as I know the subject under test rather than what could be a local function definition

Reply:
Done, had to change some things along with the function declaration, but its done and working